store/src/main/java/com/issoft/store/RandomStorePopulator.java

```diff
@@ -19,20 +18,33 @@
     Faker faker = new Faker();
 
     public Product createProduct(String product) {
-        String productName;
         if (product == FRUIT) {
-            productName = faker.food().fruit();
-        } else if (product == VEGETABLE) {
-            productName = faker.food().vegetable();
+        Product createdProduct = Product.newBuilder()
+                .setProductName(faker.food().fruit())
+                .setProductRate(faker.number().numberBetween(0, 5))
+                .setProductPrice(faker.number().numberBetween(0, 50))
+                .build();
+        return createdProduct;
+        }
+        if (product == VEGETABLE) {
+            Product createdProduct = Product.newBuilder()
+                    .setProductName(faker.food().vegetable())
+                    .setProductRate(faker.number().numberBetween(0, 5))
+                    .setProductPrice(faker.number().numberBetween(0, 35))
+                    .build();
+            return createdProduct;
         } else if (product == BOOK) {
-            productName = faker.book().title();
+            Product createdProduct = Product.newBuilder()
+                    .setProductName(faker.book().title())
+                    .setProductRate(faker.number().numberBetween(0, 5))
+                    .setProductPrice(faker.number().numberBetween(0, 45))
+                    .build();
```

Comment:
I see that the mostly the same code is used 3 times. I suggest to move the logic of create product in the private method in this class and reuse it.